src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -103,6 +103,17 @@ public void ApplyAppSettings(IEnvironment environment)
             {
                 environment.SetEnvironmentVariable(pair.Key, pair.Value);
             }
+
+            if (CorsSettings != null)
+            {
+                environment.SetEnvironmentVariable(EnvironmentSettingNames.CorsSupportCredentials, CorsSettings.SupportCredentials.ToString());
```

Comment:
I think we need a null check for SupportCredentials also.

Reply:
Do we? `SupportCredentials` is a `bool`, so it will be `false` by default, never `null`, or so I thought.